services/api/service.go

```diff
@@ -1649,6 +1649,50 @@ func (api *RelayAPI) checkFloorBidValue(opts bidFloorOpts) (*big.Int, *logrus.En
 	return floorBidValue, opts.log, true
 }
 
+type redisUpdateOpts struct {
```

Comment:
i'd change the name to be more explicit, like `redisUpdateBidOpts`

Reply:
done, thanks!